tests/foreman/ui/test_navigation.py

```diff
@@ -0,0 +1,116 @@
+# -*- encoding: utf-8 -*-
+"""Test class for Navigation UI
+
+@Requirement: Navigation
+
+@CaseAutomation: Automated
+
+@CaseLevel: Acceptance
+
+@CaseComponent: UI
+
+@TestType: Functional
+
+@CaseImportance: High
+
+@Upstream: No
+"""
+
+from robottelo.decorators import skip_if_bug_open, tier1
+from robottelo.test import UITestCase
+from robottelo.ui.locators import common_locators, menu_locators
+from robottelo.ui.session import Session
+
+
+class NavigationTestCase(UITestCase):
+    """Implements the navigation tests from UI"""
+
+    def page_objects(self):
```

Comment:
Could be a property?

```python
@property
def page_objects(self):
```

then

```python
for page in self.page_objects.values():  # without the ( )
```

Or renamed to `get_page_objects` to be used as a method `for page in self.get_page_objects()`

?